tests/gauge_path_test.cpp

```diff
@@ -214,6 +235,140 @@ void gauge_force_test(bool compute_force = true)
   delete Mom_ref_milc;
 }
 
+void gauge_loop_test()
+{
+  int max_length = 6;
+
+  setVerbosityQuda(QUDA_VERBOSE, "", stdout);
```

Comment:
should that respect the commandline option?

Reply:
Not that this is a good excuse, that what was there before, but I can patch that up.

Reply:
Addressed locally, will push soon.

Reply:
https://github.com/lattice/quda/pull/1308/commits/ccb7dd9dfe1dbb485f0d2f05456c9276a79fbafb